test/common/init/target_impl_test.cc

```diff
@@ -117,7 +103,7 @@ TEST(InitSharedTargetTestImpl, ReInitializeWhenUnavailable) {
 }
 
 // SharedTarget notifies multiple watchers.
-TEST(InitSharedTargetTestImpl, NotifyAllWatcherWhenInitialization) {
+TEST(InitSharedTargetImplTest, NotifyAllWatcherWhenInitialization) {
```

Comment:
Nit: Since you took the `Init` prefix off of the type-parameterized tests above, might as well do it here too :)